service/src/main/java/bio/terra/externalcreds/controllers/FenceAccountKeyController.java

```diff
@@ -18,42 +17,41 @@
 public record FenceAccountKeyController(
     AuditLogger auditLogger,
     ExternalCredsSamUserFactory samUserFactory,
-    FenceAccountKeyService fenceAccountKeyService,
-    HttpServletRequest request,
-    LinkedAccountService linkedAccountService)
+    FenceProviderService fenceProviderService,
+    HttpServletRequest request)
     implements FenceAccountKeyApi {
 
   @Override
   public ResponseEntity<String> getFenceAccountKey(Provider provider) {
     var samUser = samUserFactory.from(request);
-    var auditLogEvenBuilder = new AuditLogEvent.Builder()
-        .auditLogEventType(AuditLogEventType.GetServiceAccountKey)
-        .clientIP(request.getRemoteAddr());
+    var auditLogEvenBuilder =
```

Comment:
Can you fix the typo while you're changing this line? I'm pretty sure I added that 😅 
```suggestion
    var auditLogEventBuilder =
```

Reply:
Done!